tests/runner.py

```diff
@@ -648,8 +666,7 @@ def get_library(self, name, generated_libs, configure=['sh', './configure'], con
   def clear(self, in_curr=False):
     for name in os.listdir(self.get_dir()):
       try_delete(os.path.join(self.get_dir(), name) if not in_curr else name)
-    emcc_debug = os.environ.get('EMCC_DEBUG')
```

Comment:
Interesting, these are both subtly wrong (or at least not what I would expect) in the case that `os.environ['EMCC_DEBUG'] = '0'`
I would've thought `'0'` was falsy but, nope.
Probably not worth fixing in this patch (or at all?), though if we did the fix would be more like the former: `if int(os.environ.get('EMCC_DEBUG', '0'))`